scripts/cipublish

```diff
@@ -55,7 +61,7 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
             GIT_COMMIT="${GIT_COMMIT}" docker-compose \
                       -f "${DIR}/../docker-compose.yml" \
                       -f "${DIR}/../docker-compose.test.yml"\
-                      build nginx api-server tile-server spark-ingest
+                      build nginx api-server tile-server spark-ingest spark-export
```

Comment:
I don't think we need to build a container, just the jar -- the ingest container exists so we could run it in Batch before we had EMR set up, but that isn't the case. When we close https://github.com/azavea/raster-foundry/issues/1318 we should delete building the ingest container.